cekit/descriptor/resource.py

```diff
@@ -226,15 +233,16 @@ def _download_file(self, url, destination, use_cache=True):
 
 class _PathResource(Resource):
 
-    def __init__(self, descriptor, directory, **kwargs):
-        # if the path si relative its considered relative to the directory parameter
-        # it defualts to CWD, but should be set for a descriptor dir if used for artifacts
+    def __init__(self, descriptor, directory, container="", **kwargs):
```

Comment:
I would probably not add another parameter and use instead the "resource" phrase. It is used only in the log message, right?